content/spin/v2/variables.md

```diff
@@ -149,22 +163,28 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
 
 > [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/variables.html)
 
-The `variables` module has a function called `get`(https://fermyon.github.io/spin-python-sdk/variables.get).
+The `variables` module has a function called `get`(https://fermyon.github.io/spin-python-sdk/variables.html#spin_sdk.variables.get).
```

Comment:
Not your problem but I wonder why the link checker wasn't flagging the old broken link...